### ISSUE TITLE
Show "who's here"

### ISSUE THREAD

**Issue by ethanresnick** [TechAtNYU/event-checkin#11]
In an ideal world, we use checkin data to dynamically generate a visualization that attendees can see in real time (projected on a wall somewhere) that shows who's at the event—pictures, names, etc—possibly organized by skills and other api data, so attendees can find people to talk to

**Comment by AbhiAgarwal** [comment]
Definitely something that would be awesome! Once we have a checkin down I don't think this would be too hard to do (just to display names). The clustering might get a little tricky if we try match similarities of people

**Comment by AbhiAgarwal** [comment]
Also to consider how we want to present this data. Ideally it wouldn't be a list but a nice interactive demo that's neat. Maybe let someone experiment and play around with this not on the infrastructure team. Could be a cool project!

**Comment by AbhiAgarwal** [comment]
Perhaps a distinction between people who are looking for projects and people who have team-mates already. This might be tricky to do, but still important to think about. We can probably utilize another data store for this temporary information. I'll update this as I think about this problem more.

**Comment by ethanresnick** [comment]
> Perhaps a distinction between people who are looking for projects and people who have team-mates already.

Yeah, I feel like that distinction should definitely be part of it (e.g. using color to differentiate who's looking for teammates), but I don't think that only people who already have teammates/projects should be left out all together, since part of the goal could be to meet people in general (whether they have projects or not).

Unrelated: When Bob and I talked about this way back, one idea for the layout was to do a force directed graph of some sort where skills dictate the attraction/repulsion forces. Not sure if that makes sense, but just adding it for reference.

**Comment by AbhiAgarwal** [comment]
@ethanresnick I think if done correctly a force directed graph could work, but I think we can simplify it way more than a force directed graph. I think there's an easy way to present this information using the layout of the page or something similar. I'm not really sure though, I'm trying to think about how we should go about doing that. I think a force-directed graph could be easy to make, but I think it could limit how much information we can play with. I'll ask this question at the meeting, and lets see what ideas we get!

**Comment by ethanresnick** [comment]
:+1:
